Lessons/Lesson22_Basic_Stats_II_Percents.ipynb

```diff
@@ -140,7 +139,13 @@
     "id": "eyMUHGews7VZ"
    },
    "source": [
-    "CHAS is the indicator variable we used last week, where 1 indicates that the property (tract) is on the Charles River and 0 means otherwise."
+    "The `SaleCondition` column lists the condition of the house sale:\n",
+    "*   `Normal`: Normal Sale\n",
+    "* `Abnorml`: Abnormal Sale -  trade, foreclosure, short sale\n",
+    "* `AdjLand`: Adjoining Land Purchase\n",
+    "* `Alloca`: Allocation - two linked properties with separate deeds, typically condo with a garage unit\n",
+    "* `Family`: Sale between family members\n",
+    "* `Partial`: Home was not completed when last assessed (associated with New Homes)",
```

Comment:
```suggestion
    "* `Partial`: Home was not completed when last assessed (associated with New Homes)"
```

This isn't a valid notebook because of the trailing `,`. That's why the build is failing